.drone.yaml

```diff
@@ -0,0 +1,80 @@
+---
+kind: pipeline
+name: linux-amd64
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: ci
+  image: briandowns/rancher-build-base:v0.1.1
```

Comment:
Note for @cbron : is this image going in the rancher org soon?

Reply:
@nickgerace this image should be replaced with `rancher/hardened-build-base` or a different image that is maintained still.

Reply:
@rmweir: thank you for image name, I have used the latest stable `rancher/hardened-build-base:v1.14.2-amd64`

Reply:
I have changed it once again to `golang:1.16` since it was requested